micronota/tests/test_workflow.py

```diff
@@ -37,18 +38,16 @@ def setUp(self):
                 write(seq, format='fasta', into=f)
 
         self.obs_tmp = mkdtemp()
-        # mock up the db path
-        self.patcher = mock.patch('micronota.util._DB_PATH', test_dir)
-        self.patcher.start()
 
     def tearDown(self):
-        self.patcher.stop()
         rmtree(self.tmp)
         rmtree(self.obs_tmp)
 
     def test_annotate(self):
-        config = _create_config(None)
-        annotate(self.test1, 'fasta', self.obs_tmp, 'genbank', 'archaea', 1, config)
+        config = Configuration()
+        config.db_dir = self.test_dir
+        annotate(self.test1, 'fasta', self.obs_tmp, 'genbank',
+                 1, 'archaea', True, config)
```

Comment:
will add that. manually checked, looks fine.

On Sun, Mar 13, 2016 at 12:59 PM, Jamie Morton <notifications@github.com>
wrote:

> In micronota/tests/test_workflow.py
> <https://github.com/biocore/micronota/pull/54#discussion_r55942730>:
>
> >          rmtree(self.tmp)
> >          rmtree(self.obs_tmp)
> >
> >      def test_annotate(self):
> > -        config = _create_config(None)
> > -        annotate(self.test1, 'fasta', self.obs_tmp, 'genbank', 'archaea', 1, config)
> > +        config = Configuration()
> > +        config.db_dir = self.test_dir
> > +        annotate(self.test1, 'fasta', self.obs_tmp, 'genbank',
> > +                 1, 'archaea', True, config)
>
> Don't you need an assert here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/54/files#r55942730>.
>